thorlcr/activities/indexread/thindexreadslave.cpp

```diff
@@ -1471,7 +1473,10 @@ class CIndexNormalizeSlaveActivity : public CIndexReadSlaveBase
                 {
                     expanding = helper->next();
                     if (!expanding)
+                    {
```

Comment:
Like the changes in hthor, I think this code is identical to the code that was there before.

Reply:
now I know next() doesn't need a called to finishedRow, yes I think so.

Reply:
Why doesn't next call lookupBlob?